x-pack/legacy/plugins/siem/public/pages/detection_engine/components/user_info/index.tsx

```diff
@@ -205,6 +219,12 @@ export const useUserInfo = (): State => {
     }
   }, [loading, isAuthenticated, isApiAuthenticated]);
 
+  useEffect(() => {
+    if (!loading && hasEncryptionKey !== isApiEncryptionKey && isApiEncryptionKey != null) {
+      dispatch({ type: 'updatehasEncryptionKey', hasEncryptionKey: isApiEncryptionKey });
```

Comment:
nit: your other strings use camelCase, so this string would be `updateHasEncryptionKey` if you want to use camelbase based on each new word.

Reply:
bad global replace, thank you for pointing it out